en/config_vtol/vtol_quad_configuration.md

```diff
@@ -89,16 +89,6 @@ It is critical that you have properly calibrated your airspeed sensor.
 It is also important that you pick an airspeed that is comfortably above your airframes stall speed (check `FW_AIRSPD_MIN`) as this is currently not checked.
 
 
-#### Fixed Wing Permanent Stabilisation

```

Comment:
@sfuhrer FYI I just deleted this section and the one shown below for flight to remove VT_FW_PERM_STAB